src/lib/typing/payload.ts

```diff
@@ -13,13 +13,16 @@ interface PlayerPayload {
 	title: string;
 	uuid: string;
 }
+interface GuidPayload {
+	id: string;
+}
 interface MetadataPayload {
 	librarySectionType: string;
 	ratingKey: string;
 	key: string;
 	parentRatingKey: string;
 	grandparentRatingKey: string;
-	guid: string;
+	Guid?: GuidPayload[];
```

Comment:
I know it's odd, and it probably shouldn't be the case, but there's actually two separate fields in the Plex webhook payload, `guid` and `Guid` (yes, I hate this too).

The `guid` field is for the Plex guid (usually beginning with `plex://`), which is used by Plex to map an item to multiple metadata sources (e.g. LastFM, TiVo/AllMusic, MusicBrainz etc).

The `Guid` field is an array of external identifiers, useful in particular for Movie and TV libraries where users want to have access to the underlying IMDB, TVDB, and TMDB identifiers for automation purposes.

That's a long-winded way to say, rather than replacing `guid` with `Guid` in this interface, we should probably keep both around for correctness.

Reply:
Fixed in https://github.com/simonxciv/eavesdrop.fm/pull/199/commits/406066e7f598c2cf138c9c8e54d056ff480f61b1